### PR TITLE
feat(lemmy): init

### PR DESCRIPTION
**Pull request by Gandalf-the-Blue**
## 🎉 Theme for Lemmy 🎉

Lemmy is [free and open-source software](https://en.wikipedia.org/wiki/Free_and_open-source_software) for running [self-hosted](https://en.wikipedia.org/wiki/Self-hosting_(web_services)) [social news](https://en.wikipedia.org/wiki/Social_news) [aggregation](https://en.wikipedia.org/wiki/News_aggregator) and [discussion forums](https://en.wikipedia.org/wiki/Internet_forum)

## 💬 Additional Comments 💬

NA
## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Comment by nekowinston** [comment]
I'll take a look at the linter error in a bit, looks like it's failing to parse the metadata (first couple of lines of the userstyle)

File: styles/lemmy/catppuccin.user.css
Comment (isabelroses) [inline styles/lemmy/catppuccin.user.css:86]: Just quickly reading over i think the linter is failing due to this and others in the file, this is just the first i found.

```suggestion
				--bs-info-bg-subtle: @accent-color;
```

**Comment by uncenter** [comment]
> I'll take a look at the linter error in a bit, looks like it's failing to parse the metadata (first couple of lines of the userstyle)

I think the issue is the author hasn't added the data to `userstyles.yml` right?

**Comment by uncenter** [comment]
Actually it looks like Lemmy has the first letter capitalized, so if you could change that in userstyles.yml and catppuccin.user.css that would be great.

**Comment by uncenter** [comment]
Great, can you update the capitalization of Lemmy next? That should be everything you need.

**Review by uncenter — APPROVED** [review]

File: styles/lemmy/catppuccin.user.css
Comment (isabelroses) [inline styles/lemmy/catppuccin.user.css:13]:
```suggestion
@var select flavor "Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
```

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
LGTM, but knowing me i might've missed something so i'll let some not merge it right away to let others see what they think, but if nothing changes in the next 48hrs this will be merged.

File: scripts/userstyles.yml
Comment (uncenter) [inline scripts/userstyles.yml:304]: ...for the third time lol, from what I can tell the capitalization of "lemmy" should in fact be "Lemmy", so that needs to be fixed.

```suggestion
    name: Lemmy
```

File: styles/lemmy/catppuccin.user.css
Comment (uncenter) [inline styles/lemmy/catppuccin.user.css:8]: (see previous suggestion)
```suggestion
@name Lemmy Catppuccin
@namespace github.com/catppuccin/userstyles/styles/lemmy
@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/lemmy
@version 0.0.1
@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/lemmy/catppuccin.user.css
@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Alemmy
@description Soothing pastel theme for Lemmy
```

File: styles/lemmy/README.md
Comment (uncenter) [inline styles/lemmy/README.md:1]: In the future, please avoid running the generation scripts yourself. The CI/GitHub Actions will do it for you and it just means more files to look through when reviewing.

**Comment by uncenter** [comment]
From the preview images... these bubbles/pills appear to be unthemed:


Screenshot 2024-01-13 at 11 07 41 (CleanShot)

Screenshot 2024-01-13 at 11 08 17 (CleanShot)

My eyes are terrible though, correct me if I'm wrong.

**Review by uncenter — CHANGES_REQUESTED** [review]
See suggestions above.

File: scripts/userstyles.yml
Comment (uncenter) [reply scripts/userstyles.yml]: No worries! I should have just made them suggestions in the first place. Also, apologies for being passive aggressive in that comment, that was rude of me!

**Comment by isabelroses** [comment]
> The original CSS shows this to be - .text-bg-light { color: #fff !important; background-color: RGBA(var(--bs-light-rgb), var(--bs-bg-opacity, 1)) !important; } and the background-color is using RGB values, if I replace that with hex values (using the variable), it doesn't work. Can you help in how I can resolve this?

You should check out:
https://github.com/catppuccin/userstyles/blob/6ace7f8dd2f3b40bee41c4331a408f2d4635e564/styles/anilist/catppuccin.user.css#L703-L705
https://github.com/catppuccin/userstyles/blob/6ace7f8dd2f3b40bee41c4331a408f2d4635e564/styles/anilist/catppuccin.user.css#L57

what this mixin does is convert the hex values into a raw rgb data, this should be an ideal fix for this situation.

**Comment by isabelroses** [comment]
Alright looks good to merge. Thanks for your hard work.

**Pull request merged**
